tests/sls/homu/get_buildbot_cfg.py

```diff
@@ -0,0 +1,16 @@
+import imp
+import json
+import sys
+
+if __name__ == '__main__':
+    sys.path.append('/home/servo/buildbot/master/')
+
+    config = imp.load_source(
+        'config',
+        '/home/servo/buildbot/master/master.cfg'
+    )
+
+    for sched in config.c['schedulers']:
+        if sched.name == 'servo-auto':
+            out = {'builders': sched.builderNames}
+            print(json.dumps(out))
```

Comment:
Add a `return 0` after printing to indicate success and exit early. After the loop (meaning `servo-auto` was not found, print an error message to sys.stderr (e.g. `error: 'servo-auto' scheduler not found) and add a `return 1`.